RNCalendarEvents.m

```diff
@@ -30,19 +30,10 @@ @implementation RNCalendarEvents
 - (NSString *)hexStringFromColor:(UIColor *)color {
     const CGFloat *components = CGColorGetComponents(color.CGColor);
 
-    CGFloat r;
-    CGFloat g;
-    CGFloat b;
-    if(components && sizeof(components) >= 3){
-        r = components[0];
-        g = components[1];
-        b = components[2];
-    }else{
-        r = 1;
-        g = 1;
-        b = 1;
-    }
-    
+    CGFloat r = components[0];
+    CGFloat g = components[1];
+    CGFloat b = components[2];
+
```

Comment:
Are you sure about this change?
This seems more defensive programming that could be legit (I didn't check the git history to find out)